lib/rerun/notification.rb

```diff
@@ -34,6 +32,13 @@ def command
         end
       end
 
+      if s.nil? and options[:notify] == true or options[:notify] == "notify-send"
+        if (cmd = command_named('notify-send'))
+          icon_str = "--icon #{icon}" if icon
+          s = "#{cmd} -t 500 --hint=int:transient:1 #{icon_str} \"#{app_name}: #{title} -- #{body}\""
```

Comment:
right, good point. just updated the PR with a new version that does exactly that.